src/lob-address-elements.js

```diff
@@ -528,6 +530,26 @@
           config.elements.zipMsg.hide();
         }
 
+        String.prototype.format = function() {
```

Comment:
Don't change the prototype behavior for String in case the baseline customer code has its own implementation or relies on it not being present.

Reply:
Excellent point!